### DISCUSSION TITLE
Add `fuse` tag to the project

### DISCUSSION THREAD

**Discussion by cipriancraciun** [mhx/dwarfs#59] [General]
Today I was searching for FUSE-based implementations, and I observed that `dwarfs` does not list the `fuse` tag / topic.  (I know it has `fuse-filesystem`, but the former is much more popular.)

Also, perhaps it could also be useful to list it on `libfuse`'s list of file-systems at <https://github.com/libfuse/libfuse/wiki/Filesystems>.

**Comment by mhx** [comment]
Thanks, I've just done both!
